admin/common_auth/models.py

```diff
@@ -39,14 +39,14 @@ class MyUser(AbstractBaseUser, PermissionsMixin):
         unique=True,
     )
 
-    first_name = models.CharField(max_length=30, blank=True)
-    last_name = models.CharField(max_length=30, blank=True)
+    first_name = models.CharField(max_length=30, blank=False)
+    last_name = models.CharField(max_length=30, blank=False)
     is_active = models.BooleanField(default=True)
     is_admin = models.BooleanField(default=False)
     is_staff = models.BooleanField(default=False)
     date_joined = models.DateTimeField(default=datetime.now)
```

Comment:
Django has a way to [do this on add](https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.DateField.auto_now_add). If you're migrating older data you'll have to decide what to do about the old ones.